src/20231106_AaveV3Ethereum_AaveFundingUpdates/AaveFundingUpdates.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Aave Funding Updates (part 2)"
+author: "TokenLogic, efecarranza.eth"
+discussions: "https://governance.aave.com/t/arfc-aave-funding-update/15194"
+---
+
+## Simple Summary
+
+This AIP aims to consolidate Aave DAO's stablecoin holdings. It is divided into different parts that are detailed below. This is part two and it consists of depositing USDC into Aave v3 Ethereum, as well as migrating the v2 position to v3 in Ethereum. Also depositing USDT into Aave v2 Ethereum and swapping DAI for GHO.
+
+## Motivation
+
+This proposal intends to ensure the DAO is well capitalised.
+
+The DAO has sufficient USDC, DAI and USDT deposited across the various instances of Aave Protocol to support existing streams and the AURA purchase 3.
```

Comment:
Revised Motivation Section:

This AIP (Part B) when implemented ensures the Aave Protocol is sufficiently funded to support service providers receiving DAI and USDT. 

The funds transferred from Polygon to Ethereum by the Aave Funding Part A AIP are to be deposited into Aave v2 and v3 as required and DAI swapped to GHO. In addition, with no USDC stream drawing from Aave v2 on Ethereum, all USDC is to be withdrawn from v2 and deposited into v3.